benchmark/sample/expression.ts

```diff
@@ -633,7 +633,7 @@ function parseParenItem(): void {
 // argument to parseSubscripts to prevent it from consuming the
 // argument list.
 function parseNew(): void {
-  parseIdentifier();
+  expect(tt._new);
```

Comment:
Haha, this one is just a sample TS file for one of the benchmarks, never actually executed, but sure.